tests/__init__.py

```diff
@@ -78,12 +105,31 @@ def test_b(self):
 
     Regular test cases will be run in usual way, while async ones - inside
     event loop.
+
+    A test class may contain special setup / teardown methods, which will
+    be executed before and after running of each test case::
+
+        class TestSnippets(unittest.TestCase, metaclass=AIOTestMeta):
+
+            def setup(self):
+                self.app = create_app(DEFAULT_CONF)
+
+            def teardown(self):
+                loop = asyncio.get_event_loop()
```

Comment:
Hm.. What do you think about making it more event-loop agnostic?

It seems we can:

* Check in decorator whether `setup`/`teardown` is a coroutine function.
* If so, run them in newly created event loop.
* If not, run them as regular methods.

In that case it could be rewritten into something like:

```python
def setup(self):
    self.app = create_app(DEFAULT_CONF)

async def teardown(self):
    await self.app['db'].snippets.drop()
```

What do you think?